README.md

```diff
@@ -1,8 +1,155 @@
-# nJwt
+# nJwt - JWTs for Node.js
+
+"Nin-Jot" /ˈnɪn.dʒɑt/
 
 [![NPM Version](https://img.shields.io/npm/v/njwt.svg?style=flat)](https://npmjs.org/package/njwt)
 [![NPM Downloads](http://img.shields.io/npm/dm/njwt.svg?style=flat)](https://npmjs.org/package/njwt)
 
-Node.js JWT support
+This Node.js library allows you create and verify JWTs.  We've made the process
+incredibly simple by choosing secure defaults for you, while still allowing you
+modify the creation and verification steps if desired.
+
+#### Creating Secure, Signed JWTs
+
+JWTs expect "claims", they a set of assertions about who the user is and what
```

Comment:
Yeah, it looks like they changed the spec since JJWT was created.  I need to read the JWS spec to confirm, but it appears that string payloads are no longer specified.